build/setScala2.12.sh

```diff
@@ -0,0 +1,9 @@
+sed -i -e "s#geomesa-geoserver</art#geomesa-geoserver_2.12</art#g" \
```

Comment:
how hard would it be to make this dynamically find all the dirs/poms? maybe that is overkill... depends how long we have to keep this around i guess. if not, we'd have to remember to update this script when adding any additional modules.

Reply:
Yeah, this is short-term fix.  I think we can remember to make a kludge for any new modules in the 3.x line.  *shrugs*